term_solaris_ioctl.go

```diff
@@ -0,0 +1,37 @@
+// +build solaris
+
+// Copyright 2017 Attila Fülöp <attila@fueloep.org>
```

Comment:
There's been a few people now who've contributed now and haven't left their name, instead of individual copyright lines in each file, do you think it would be fine to instead place your name in a `CONTRIBUTORS` file?